### PR TITLE
Updated documented standards to current standards.

### PR DESCRIPTION
**Pull request by nesl247**

### CODE REVIEW

File: standards/commit.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ You SHOULD be committing frequently. This does not mean you need to commit every
 ### End of Day
 You MUST commit and push all of your changes before you leave for the end of the day. If something happens to your laptop or you, the company should not lose access to any changes that were in progress. If you have a work in progress commit, you MUST prefix your commit message with `[WIP]`. An example is `[WIP] Added delete action to UserController.`.
 
+## Branch History
+You SHOULD squash all related changes to a single commit. Commits such as `[WIP]`, `Added return type`, etc.,
+SHOULD be combined so as to keep the history clean. While developing it is fine to have these, but once a PR is submitted,
+the branch is considered to be ready for production. We DO NOT want to pollute fixed branches such as master with commits
```
Comment (klaussilveira) [inline standards/commit.md]: I don't think that "DO NOT" should be in caps here, it conflicts with the clear keywords we've defined at the top of the document.

Comment (nesl247) [inline standards/commit.md]: Fixed

**Comment by klaussilveira** [comment]
:+1:

**Comment by damian-af** [comment]
👍

**Pull request merged**